CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Basedmypy Changelog
 
 ## [Unreleased]
+### Enhancements
+- Render inferred literal without `?`
```

Comment:
isn't this more of a bug fix than an enhancement

Reply:
Who knows tbh, but good point.